kriswallsmith/buzz/0.17/config/packages/buzz.yaml

```diff
@@ -0,0 +1,35 @@
+# Read more about client options at: https://github.com/kriswallsmith/Buzz/blob/master/doc/client.md
+services:
+    _defaults:
+        public: false
+
+    # Tell the auto wire system to use "FuleGetContents" client as default client.
+    Buzz\Client\BuzzClientInterface: '@Buzz\Client\FileGetContents'
+
+    # Clients
+    Buzz\Client\Curl:
```

Comment:
I would not use class names as service ids for the client implementation, so that only the interface can be autowired.

Reply:
But what if you really want a specific client? Say that for some requests you want the MultiCurl and for some you want normal Curl... 

Is there any performance issues in using class names as I do?